### PR TITLE
[10.0][MIG] port sale_product_multi_add to 10.0

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
Port PR https://github.com/OCA/sale-workflow/pull/317 on V8.0 to V10.0

### CODE REVIEW

**Review by antoniocanovas — APPROVED** [review]
Ok Functionality

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍

File: sale_product_multi_add/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Usage
+=====
+
+This module add a button on sale order to import several products on your current sale order.
+A wizard opens and allows the user to search and select products.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{sale-workflow}/{10.0}
```
Comment (tafaRU) [inline sale_product_multi_add/README.rst]: please avoid brackets

File: sale_product_multi_add/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (tafaRU) [inline sale_product_multi_add/README.rst]: please remove this commented part.

File: sale_product_multi_add/views/sale_view.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (tafaRU) [inline sale_product_multi_add/views/sale_view.xml]: you could remove `data` tag

File: sale_product_multi_add/wizard/sale_import_products.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# © 2016 Cédric Pigeon, ACSONE SA/NV (<http://acsone.eu>)
```
Comment (tafaRU) [inline sale_product_multi_add/wizard/sale_import_products.py]: please replace © by Copyright, see https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L2 as reference

File: sale_product_multi_add/wizard/sale_import_products_view.xml [hunk 5]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (tafaRU) [inline sale_product_multi_add/wizard/sale_import_products_view.xml]: same as above

**Review by tafaRU — CHANGES_REQUESTED** [review]

File: sale_product_multi_add/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 Cédric Pigeon, ACSONE SA/NV (<http://acsone.eu>)
```
Comment (tafaRU) [inline sale_product_multi_add/__manifest__.py]: please replace © by Copyright, see https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L2 as reference

**Review by tafaRU — COMMENTED** [review]
In addition, folder `wizard` should be renamed to `wizards` according to https://github.com/OCA/maintainer-tools/tree/master/template/module/wizards

**Review by tafaRU — COMMENTED** [review]
The rest LGTM, I already tested it on runbot.
Thanks!

**Comment by Cedric-Pigeon** [comment]
@tafaRU Remarks integrated.

**Comment by Cedric-Pigeon** [comment]
@tafaRU Hi, if it is ok for you can you please approve this PR to merge it. Thanks

**Comment by bealdav** [comment]
@tafaRU any news here ? Thanks

**Comment by rafaelbn** [comment]
Please @bealdav could you add you review?

Please @Cedric-Pigeon could you squash some commits?

I will merge ASAP

**Comment by Cedric-Pigeon** [comment]
@rafaelbn I squashed commits. Thanks

**Review by tafaRU — APPROVED** [review]

**Pull request merged**